insights/tests/parsers/test_installed_rpms.py

```diff
@@ -539,3 +539,19 @@ def test_vmaas():
     assert isinstance(rpm, InstalledRpm)
     assert rpm.version == "5.2.2"
     assert rpm.release == "1.el7"
+
+
+def test_container_installed_rpms():
+    rpms = ContainerInstalledRpms(context_wrap(RPMS_PACKAGE))
+    assert rpms.get_min('openldap').package == 'openldap-2.4.23-31.el6'
+    pkg_rpm = rpms.packages['openssh-server'][0]
+    rpm = InstalledRpm.from_package(pkg_rpm.package)
+    assert rpm.package == 'openssh-server-5.3p1-104.el6'
+    assert pkg_rpm.package == 'openssh-server-5.3p1-104.el6'
+    assert rpm == pkg_rpm
+    assert rpm.epoch == '0'
```

Comment:
Better add `Container` related items to the test. e.g. the `.engine`, `.image`, and `.container_id`, etc.